src/server/controllers/user.js

```diff
@@ -0,0 +1,31 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const express = require('express');
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+const secret = "kjfjksn"
+
+const postRegisterDetails = async (req, res) => {
+    try {
+        const { email, username, password } = req.body;
+        const hashedPassword = await bcrypt.hash(password, saltRounds);
+        const newUser = await prisma.user.create({
+            data: {
+                email: email,
+                username: username,
+                password: hashedPassword
+            }
```

Comment:
this will need more data added once the user model has been fleshed out more

Reply:
So far this is all that the user will fill in? What else will we need?

Reply:
Yeah youre right, I guess stuff like posts/comments/ratings are optional on the model so dont need created.